sections/05_treat.md

```diff
@@ -162,45 +162,193 @@ feature selection and construction.*
+substituting discrete operations in the fingerprinting algorithm with operations
+in a neural network, producing a real-valued feature vector instead of a bit
+vector.  Other approaches offer trainable networks that can in theory learn
+chemical feature representations that are optimized for a particular prediction
+task.   Lusci et al adapted recursive neural networks for directed acyclic
+graphs for undirected molecular graphs by creating an ensemble of directed
+graphs in which one atom is selected as the root node [@tag:Lusci2013_rnn].  A
+single feature vector is obtained by summing over all feature vectors for all
+directed graphs in the ensemble.  Graph convolutions on undirected molecular
+graphs have eliminated the need to enumerate artificial directed graphs,
+learning feature vectors for atoms that are a function of the properties of
+neighboring atoms and local regions on the molecular graph
+[@tag:Kearnes2016_graph_conv @tag:AltaeTran2016_one_shot]. `TODO: add more
+details on how graph convolutions work?`
+
+Advances in chemical representation learning have not been limited to
+graph-based neural networks.  The simplified molecular-input line-entry system
+(SMILES) is a standard way to transform a chemical into string-based
+representation.  A SMILES-to-SMILES autoencoder learns a continuous latent
+feature space for chemicals [@tag:Gomezb2016_automatic].  In this learned
+continuous space is it possible to train some types of supervised learning
+algorithms or interpolate between continuous representations of chemicals in a
+manner that is not possible with discrete (e.g. bit vector or string) features.
+A drawback is that not all SMILES strings produced by the autoencoder's decoder
+correspond to valid chemical structures. `TODO: could mention GANs here`
+Altae-Tran et al developed a one-shot learning network
+[@tag:AltaeTran2016_one_shot] to address the reality that most practical
+chemical screening studies are unable to provide the thousands or millions of
+training compounds that are needed to train larger multitask networks.  Using
+graph convolutions to featurize chemicals, the network learns an embedding from
+compounds into a continuous feature space such that compounds with similar
+activities in a set of training tasks have similar embeddings.  The approach is
+evaluated in an extremely challenging setting where the embedding is learned
+from a subset of prediction tasks (e.g. activity assays for individual proteins)
+and only one to ten labeled examples are provided as training data on a new
+task.  On Tox21 targets, even when trained with _one_ task-specific active
+compound and _one_ inactive compound, the model is able generalize reasonably
+well because it has learned an informative embedding function from the related
+tasks. Random forests, which cannot take advantage of the related training
+tasks, trained in the same setting achieve near random performance.  Despite the
```

Comment:
achieve sounds very positive here - maybe `do not best random performance` or something that makes it clear that near random is not good.